robot_smach_states/examples/manipulation/grasp_benchmark/grasping_height_test.py

```diff
@@ -0,0 +1,77 @@
+#! /usr/bin/env python
+
+# System
+from collections import namedtuple
+
+# ROS
+import PyKDL as kdl
+import rospy
+
+# TU/e Robotics
+from robot_smach_states.manipulation import Grab
+from robot_smach_states.navigation import NavigateToSymbolic
+from robot_smach_states.util.designators import EntityByIdDesignator, UnoccupiedArmDesignator
+
+# Robot Skills
+from robot_skills import arms
+from robot_skills.hero_parts.hero_arm import HeroArm
+from robot_skills.get_robot import get_robot_from_argv
+from robot_skills.util.kdl_conversions import FrameStamped
+
+DZ = 0.1
+Z_MAX = 1.2
+
+X_ENTITY = 3.2
+Y_ENTITY = 1.55
+
+Result = namedtuple("Result", ["height", "result", "duration"])
+
+if __name__ == "__main__":
+
+    rospy.init_node("test_grasping")
+
+    robot = get_robot_from_argv(1)
```

Comment:
Please add some instructions so that people know how to use the test

Reply:
Added script docstring in 7702914abf384c911ce846bb5598e7949468e2b2